astropy/coordinates/builtin_frames/icrs_cirs_transforms.py

```diff
@@ -17,12 +17,62 @@
 from ... import _erfa as erfa
 
 from .icrs import ICRS
+from .itrs import ITRS
 from .gcrs import GCRS
 from .cirs import CIRS
 from .hcrs import HCRS
 from .utils import get_jd12
 
 
+"""
+rotational velocity of Earth in radians per (UT1) second
```

Comment:
This confused me at first, since if it really was per UT1 seconds, it should just be 2 pi / 24 / 3600. But the rotational velocity given is actually per TAI second. Maybe rephrase the docstring
```
Rotational velocity of the Earth. In UT1 seconds, this would be 2 pi / (24 * 3600),
but we need the value in SI seconds.
```